experiment/build/gcb_build.py

```diff
@@ -40,77 +39,29 @@
 logger = logs.Logger('builder')  # pylint: disable=invalid-name
 
 
-def build_base_images() -> Tuple[int, str]:
+def build_base_images():
     """Build base images on GCB."""
-    return _build(get_build_config_file('base-images.yaml'), 'base-images')
+    _build(get_build_config_file('base-images.yaml'), 'base-images')
 
 
 def build_coverage(benchmark):
     """Build coverage image for benchmark on GCB."""
-    if benchmark_utils.is_oss_fuzz(benchmark):
-        _build_oss_fuzz_project_coverage(benchmark)
-    else:
-        _build_benchmark_coverage(benchmark)
-
-
-def _build_benchmark_coverage(benchmark: str) -> Tuple[int, str]:
-    """Build a coverage build of |benchmark| on GCB."""
     coverage_binaries_dir = exp_path.filestore(
         build_utils.get_coverage_binaries_dir())
     substitutions = {
         '_GCS_COVERAGE_BINARIES_DIR': coverage_binaries_dir,
         '_BENCHMARK': benchmark,
     }
     config_file = get_build_config_file('coverage.yaml')
-    config_name = 'benchmark-{benchmark}-coverage'.format(benchmark=benchmark)
-    return _build(config_file, config_name, substitutions)
-
-
-def _build_oss_fuzz_project_fuzzer(benchmark: str,
-                                   fuzzer: str) -> Tuple[int, str]:
-    """Build a |benchmark|, |fuzzer| runner image on GCB."""
-    substitutions = {
-        '_BENCHMARK': benchmark,
-        '_FUZZER': fuzzer,
-    }
-    config_file = get_build_config_file('oss-fuzz-fuzzer.yaml')
-    config_name = 'oss-fuzz-{benchmark}-fuzzer-{fuzzer}'.format(
-        benchmark=benchmark, fuzzer=fuzzer)
-
-    return _build(config_file, config_name, substitutions)
-
-
-def _build_benchmark_fuzzer(benchmark: str, fuzzer: str) -> Tuple[int, str]:
-    """Build a |benchmark|, |fuzzer| runner image on GCB."""
-    # See link for why substitutions must begin with an underscore:
-    # https://cloud.google.com/cloud-build/docs/configuring-builds/substitute-variable-values#using_user-defined_substitutions
-    substitutions = {
-        '_BENCHMARK': benchmark,
-        '_FUZZER': fuzzer,
-    }
-    config_file = get_build_config_file('fuzzer.yaml')
-    config_name = 'benchmark-{benchmark}-fuzzer-{fuzzer}'.format(
-        benchmark=benchmark, fuzzer=fuzzer)
-    return _build(config_file, config_name, substitutions)
-
-
-def _build_oss_fuzz_project_coverage(benchmark: str) -> Tuple[int, str]:
-    """Build a coverage build of OSS-Fuzz-based benchmark |benchmark| on GCB."""
-    coverage_binaries_dir = exp_path.filestore(
-        build_utils.get_coverage_binaries_dir())
-    substitutions = {
-        '_GCS_COVERAGE_BINARIES_DIR': coverage_binaries_dir,
-        '_BENCHMARK': benchmark,
-    }
-    config_file = get_build_config_file('oss-fuzz-coverage.yaml')
-    config_name = 'oss-fuzz-{benchmark}-coverage'.format(benchmark=benchmark)
-    return _build(config_file, config_name, substitutions)
+    config_name = '{benchmark}-coverage'.format(benchmark=benchmark)
```

Comment:
Shouldn't this be benchmark-{benchmark}-coverage

Reply:
Done.